var/spack/repos/builtin/packages/lm-sensors/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class LmSensors(MakefilePackage):
+    """The lm-sensors package provides user-space support for the
+    hardware monitoring drivers in Linux. """
+
+    homepage = "https://github.com/groeck/lm-sensors/"
+    url = "https://github.com/groeck/lm-sensors/archive/V3-4-0.tar.gz"
+
```

Comment:
shouldn't there be `bison`, `flex` and `rrd` dependencies of type `build`?
https://github.com/lm-sensors/lm-sensors/blob/master/INSTALL

`perl` and `rrdtool` seem to be `run` dependencies as well

Reply:
I guess that my testing got those build dependencies from my system.  I can add them.

I would prefer to have perl as an optional run dependency since it is only needed for running one script that is installed by lm-sensors.  Would this be OK?

The rrd and rrdtool is needed for the (optional) sensord and web-based graphing functionality, but I don't think people should be running system services that are installed with spack.

Reply:
> perl as an optional run dependency since it is only needed for running one script

is it a very common/central one? if you say it's just user-candy and not central I think it's fine to skip `perl` as a runtime dep

> The rrd and rrdtool is needed for the (optional) sensord ...

makes sense!

Reply:
I noticed that either bison or flex already requires perl, so adding it as a run dependency doesn't increase the number of things to install.